components/TicketsSection.js

```diff
@@ -88,6 +88,18 @@ function Button ({ children, href }) {
           background: #00D8FF;
           color: #FFF;
         }
+        .button_full {
```

Comment:
Instead of another class with most of the CSS attributes are the same, I suggest you to use modifier class instead. E.g. `.button.full`. It will reduce lines of code by being [DRY](https://en.wikipedia.org/wiki/Don%27t_repeat_yourself).